### DISCUSSION TITLE
Discussion of updates

### DISCUSSION THREAD

**Discussion by JesusChapman** [Yisus7u7/termux-desktop-xfce#18] [General]
Here you can comment on each update or change of the project

**Comment by JesusChapman** [comment]
@suhan-paradkar
I e configured the use of mate-terminal, it is very nice and clearer, but.... 

You have an error with the composition. It's up to you to use it without transparency. 

This update and themes were going to do for termux-desktop-mate, but it is still incomplete and does not have to configure themes, I made that I decided to postpone it

**Reply by creepy-pasta101** [reply]
Thats fine... also... there are a few things missing in mate desktop...
mate-control-center currently has a few issues as it needs polkit-gobject... gobject-introspection is present but polkit isn't....
There are NO mate themes present in x11-repo...
So it's okay to postpone its inclusion

**Reply by JesusChapman** [reply]
WellYes, but what do you explain about the mate-terminal bug?

**Reply by creepy-pasta101** [reply]
I guess that would need to be fixed

**Reply by JesusChapman** [reply]
Well, yes

**Comment by JesusChapman** [comment]
![Screenshot_20210802-163519](https://user-images.githubusercontent.com/64093255/128033089-d7ce69ab-71b5-4166-89bd-cab8b18628ff.png)
![Screenshot_20210802-163457](https://user-images.githubusercontent.com/64093255/128033096-f47897f2-578c-4bc9-8ac5-9025ee16d33c.png)

A big upgrade is coming, and the best part is that it will now take up less storage space than before!...

**Reply by creepy-pasta101** [reply]
That would be great... Also.. please instruct users to perform a shallow clone than a regular one.... That would reduce data usage...

I mean.. use 

`git clone https://xyz/abc --depth=1`

Instead of just 

`git clone https://xyz/abc`

**Reply by JesusChapman** [reply]
Thanks

**Comment by JesusChapman** [comment]
And the new version is ready, I'll completely update the README then, I'm sleepy, then I'll upload some deb packages that I did in the releases section.
